lib/ansible/modules/cloud/amazon/iam_cert.py

```diff
@@ -269,10 +281,23 @@ def main():
     cert_chain = module.params.get('cert_chain')
     dup_ok = module.params.get('dup_ok')
     if state == 'present':
-        cert = open(module.params.get('cert'), 'r').read().rstrip()
```

Comment:
You'll also note I updated the else statement to correctly reflect the vars used later.
eg. cert_chain not chain.